.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,134 @@
+name: Benchmark
+on:
+    workflow_dispatch:
+        inputs:
+            exclude-slow-benchmarks:
+                description: 'Exclude slow benchmarks'
+                required: true
+                type: boolean
+                default: false
+            exclude-unstable-benchmarks:
+                description: 'Exclude unstable benchmarks'
+                required: true
+                type: boolean
+                default: false
+            exclude-release-benchmarks:
+                description: 'Exclude release benchmarks'
+                required: true
+                type: boolean
+                default: false
+
+jobs:
+    get-exclude-dirs:
+        name: Get exclude directories
+        runs-on: ubuntu-latest
+        outputs:
+            exclude-dirs: ${{ steps.get-exclude-dirs.outputs.exclude-dirs }}
+        steps:
+            - uses: actions/checkout@v4
+
+            - id: get-exclude-dirs
+              uses: ./.github/actions/get_exclude_dirs
+              with:
+                  exclude-slow: ${{ github.event.inputs.exclude-slow-benchmarks == true }}
```

Comment:
Just throw the variables in right? Just pass the boolean along

Reply:
done